### PR TITLE
Import docs and small improvements for card styling

### PR DESCRIPTION
**Pull request by kepano**
- Add new `list-cards` CSS helper to turn lists into cards
- Update a few pages including Home and Core Plugins to use card style
- Add new migration guides
- Add Importer plugin
- Alphabetize the Glossary page

### CODE REVIEW

File: en/Import notes/Import from Bear.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+## Import your Bear data into Obsidian
+
+You will need the official Obsidian [[Importer]] plugin, which you can [install here](obsidian://show-plugin?id=obsidian-importer).
+
+1. Open **Settings**.
+2. Go to **Community Plugins** and [install Importer](obsidian://show-plugin?id=obsidian-importer).
+3. Enable the Importer plugin
+4. Open the **Importer** plugin using the command palette or ribbon icon.
+5. Under **File format** choose **Bear (.bear2bk)**
```
Comment (ericaxu) [inline en/Import notes/Import from Bear.md:26]: missing period

File: en/Import notes/Import from Bear.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+4. Click **Export notes**.
+5. You should now have one `.bear2bk` file that contains all your notes.
+
+## Import your Bear data into Obsidian
+
+You will need the official Obsidian [[Importer]] plugin, which you can [install here](obsidian://show-plugin?id=obsidian-importer).
+
+1. Open **Settings**.
+2. Go to **Community Plugins** and [install Importer](obsidian://show-plugin?id=obsidian-importer).
+3. Enable the Importer plugin
```
Comment (ericaxu) [inline en/Import notes/Import from Bear.md:24]: missing period

File: en/Import notes/Import from Bear.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+4. Open the **Importer** plugin using the command palette or ribbon icon.
+5. Under **File format** choose **Bear (.bear2bk)**
+6. Select the location of your Bear backup file
```
Comment (ericaxu) [inline en/Import notes/Import from Bear.md:27]: missing period

File: en/Import notes/Import from Bear.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+7. Click **Import** and wait until import is complete
```
Comment (ericaxu) [inline en/Import notes/Import from Bear.md:28]: missing period

File: en/Import notes/Import from Evernote.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+## Export your data from Evernote
+
+Obsidian uses Evernote's export format `.enex` files.
+
+You can find Evernote's instructions for exporting your data [on Evenote's website](https://help.evernote.com/hc/en-us/articles/209005557-Export-notes-and-notebooks-as-ENEX-or-HTML). This method allows you to export entire notebooks in the desktop client.
+
+1. Go to the Notebooks screen
+2. Click on **More actions** (`•••` icon) and choose **Export Notebook...**
+3. Select **ENEX** as the file format
```
Comment (ericaxu) [inline en/Import notes/Import from Evernote.md:14]: missing periods from 1 to 3

File: en/Getting started/Glossary.md [hunk 6]
```diff
@@ -1,104 +1,105 @@
...
+## hotkey
 
-An **attachment** is an [[Accepted file formats|accepted file format]] that was created outside of the vault and added later.
+A **hotkey** is a keyboard shortcut for a [[#command]]. See also [[Custom hotkeys]] and [[Custom hotkeys|How to/Use hotkeys]].
 
-## plugin
+## link
 
-A **plugin** extends Obsidian with additional features.
+A **link** references another note, file, or document. An [[Internal links|internal link]] references a file inside a vault. An [[Basic formatting syntax#External links|external link]] references a document outside the vault.
```
[the remark this answers is not in the document]
Comment (kepano) [reply en/Getting started/Glossary.md]: I took another pass at it, removing the term document

**Comment by kepano** [comment]
> Did you test this on local publish or should I test it?

Yes I did try it locally and everything seems to be working

**Pull request merged**